test/AutoRest.TestServer.Tests/ConvenienceMethodTests.cs

```diff
@@ -238,10 +238,13 @@ public void NoConvenienceOptionalBodyScenario()
         [Test]
         public void SetKeepNonOverloadableProtocolSignature()
         {
-            var method = typeof(MixApiVersionClient).GetMethod("Delete");
+            var getPetsClientMethod = typeof(MixApiVersionClient).GetMethod("GetPetsClient");
```

Comment:
Just curious, whey the emitter change would impact autorest tests?

Reply:
This is typespec code, not autorest, so expected. We should move these tests to some typespec places later.